api/src/backend/api/v1/views.py

```diff
@@ -3235,15 +3235,50 @@ def destroy(self, request, *args, **kwargs):
 @extend_schema_view(
     list=extend_schema(
         tags=["Compliance Overview"],
-        summary="List compliance overviews for a scan",
-        description="Retrieve an overview of all the compliance in a given scan.",
+        summary="List compliance overviews",
+        description=(
+            "Retrieve an overview of all compliance frameworks. "
+            "If scan_id is provided, returns compliance data for that specific scan. "
```

Comment:
```suggestion
            "If `scan_id` is provided, returns compliance data for that specific scan. "
```

I believe that it will be better if parameters are wrapped with `` since it make it look better in Redoc

Reply:
Totally, I'll note it down to fix these kinds of details in a style PR, thanks!